cdc/processor/pipeline/sorter.go

```diff
@@ -211,8 +220,28 @@ func (n *sorterNode) Receive(ctx pipeline.NodeContext) error {
 					zap.Uint64("oldResolvedTs", oldResolvedTs))
 			}
 			atomic.StoreUint64(&n.resolvedTs, rawKV.CRTs)
+
+			if resolvedTs > n.barrierTs &&
+				!redo.IsConsistentEnabled(n.replConfig.Consistent.Level) {
```

Comment:
Don't need to check whether redo log is enabled here, since it doesn't break anything when redo log is enabled.

Reply:
I think it may cause deadlock, because when redolog is enable, resolved ts is calculated from sink. If sorter cuts resolved ts, sink can never advance its resolved ts.

Reply:
sink can forward resolved ts to barrier ts at least, why deadlock happens